Xamarin.Essentials/Screenshot/Screenshot.ios.tvos.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.IO;
+using System.Threading.Tasks;
+using UIKit;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Screenshot
+    {
+        internal static bool PlatformCanCapture => UIScreen.MainScreen != null;
+
+        static string GetTempFileName()
+        {
+            var tempFileName = Path.ChangeExtension(Path.Combine(FileSystem.CacheDirectory, Path.GetTempFileName()), ".png");
```

Comment:
To be consistent, we should probably use `Path.Combine(FileSystem.CacheDirectory, Path.GetTempFileName());` like Android. Also, as it is a single line, just put it in the `PlatformCaptureAsync` method.